res/style.css

```diff
@@ -776,6 +776,8 @@ body, #root, .profileViewer {
 
 .react-contextmenu.react-contextmenu--visible {
   display: block;
+  max-height: 450px;
+  overflow-y: auto;
 }
```

Comment:
On my system (linux, but I suspect Windows would be similar, as they're both platforms with non-overlay scrollbars), I think we miss some padding on the right, see: 
![capture d ecran de 2018-05-14 18-40-22](https://user-images.githubusercontent.com/454175/40011060-f13927d4-57a6-11e8-8e46-b6ba1d8826e1.png)

When inspecting:
![context-menu-issue](https://user-images.githubusercontent.com/454175/40011234-76098f94-57a7-11e8-8f82-09aefbbe352f.png)

I see we actually have some padding but the text is escaping it. When comparing before/after your patch, I see the menu's width is the same. I'm not sure where it comes from as I don't see a fixed size anywhere, so we may have an issue in Gecko in how the width is computed. If so I'm fine landing this as is but if you can work on a simpler testcase and file a bug on bugzilla this would be nice !

Reply:
There might be one already, as this is definitely a known issue. (I recall discussing this a few months ago). Will follow up as soon as I know.

Reply:
+1 on auto.

I'm okay with the padding, but would prefer to file a follow up big for this. Those small, defined bugs are very valuable to new contributors. So this would be a great use case for me. :)

Reply:
I'm not a big fan of follow-ups for very small changes especially when they're related to the current PR. This sounds to be a lot of overhead.

What I mean I'm really OK to do such bugs or follow-ups when we find unrelated issues or changes to do while working on a PR. This is not the case here: the change is completely related to this PR.

But I don't want to block on this so please proceed if you think it's valuable. I may be wrong too :-)

Reply:
I truly think the small bugs are the ones that get more people into the project as all `good first bugs` are more work or some of them more complex. So this could be a really good first step towards opening up a little bit more.

As I really want that patch to land, I did it by myself as its faster this was as discussing if its a good way to go or not. :)